ui/client/documents/upload/FileList.js

```diff
@@ -13,10 +13,9 @@ import Paper from '@material-ui/core/Paper';
 import RadioGroup from '@material-ui/core/RadioGroup';
 import Radio from '@material-ui/core/Radio';
 
-/**
- *
```

Comment:
Hey- is there a particular reason you're removing these? I know the jsdoc-like comments are empty right now, but I'd rather add some description here than remove the jsdoc placeholder.

To share my reasoning: It both adds a description, but I also take advantage that it already separates symbol/function definitions in a way that gets me more room between each other while going through the file:

<img src="https://github.com/jataware/dojo-stack/assets/1967061/ddeffacc-2e15-4f0e-b6be-271a83ddabe5" width="50%" />

<img src="https://github.com/jataware/dojo-stack/assets/1967061/840d8af0-b471-4b57-8d82-ab65f128bab3" width="50%" />

but of course that's just a cherry on top- the end-goal is to eventually for me (or anyone else that runs into this!) to fill in or add whatever's useful for them (if there isn't anything already).

No big deal. Curious to hear your thoughts on my approach to fn padding and docstrings and/or if it bothers you that they're sometimes empty.

Reply:
I use an editor that runs eslint and shows me the errors, so I generally clean up small things that throw linter errors when I'm in a file. Empty comments are one such thing. I don't feel strongly about them either way, though I'd both lean more towards separating things out into separate files to make them more legible and just adding a comment when there is actually one to be added, rather than preemptively putting space in for it. 

(and separating things out into different files is a totally subjective, not at all all-the-time thing, so even my feelings on that are complicated)

But like I said, I don't feel strongly about it. I do like having a linter and style guidelines, as I find it helps me focus and not need to make little decisions about minor inconsequential things. Plus the linter does catch actual issues (such as missing dependencies in hooks). But that's a whole separate discussion 😄. 

All of which is to say, I'm happy to stop deleting those from the top of functions/components. I'll look into adding an exception to the linter that will make it stop complaining about those specific empty comments, rather than all empty comments... And feel free to let me know if any of my other linting-inspired cleanup when I'm in a file changes something you would rather not change!

Reply:
Adding them back! And Eslint was just throwing a fit because of a lack of spacing after the comment started, so I added an exception for that.